package.json

```diff
@@ -19,10 +21,11 @@
   },
   "dependencies": {
     "debug": "^4.1.1",
-    "winston": "^3.2.1"
+    "winston": "3.12"
```

Comment:
this won't be needed anymore once #60 is merged

Reply:
@hparra please revert [b6fdb40](https://github.com/adobe/aio-lib-core-logging/pull/57/commits/b6fdb403415377972fea64f31d1e245d8714c6cf) so we can pull this in. I've merged the fix from master

Reply:
@shazron done